caas/kubernetes/provider/provider.go

```diff
@@ -31,15 +31,15 @@ import (
 type kubernetesEnvironProvider struct {
 	environProviderCredentials
 	cmdRunner          CommandRunner
-	builtinCloudGetter func(CommandRunner) (cloud.Cloud, jujucloud.Credential, string, error)
+	builtinCloudGetter func(CommandRunner) (cloud.Cloud, error)
 	brokerGetter       func(environs.OpenParams) (caas.ClusterMetadataChecker, error)
 }
 
 var _ environs.EnvironProvider = (*kubernetesEnvironProvider)(nil)
 var providerInstance = kubernetesEnvironProvider{
 	environProviderCredentials: environProviderCredentials{
 		cmdRunner:          defaultRunner{},
-		builtinCloudGetter: attemptMicroK8sCloud,
+		builtinCloudGetter: attemptMicroK8sCredential,
```

Comment:
the name change doesn't appear right to me
we are assigning to "buildinCloudGetter" so "attemptMicroK8sCloud" seems correct

Reply:
Done & Done